.buildkite/scripts/steps/fleet/install_all_packages.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
I don't think there's a way to tell from the Github UI... is this file `chmod +x`'d?

Reply:
Nope!

Reply:
Oh, if you "view file" from the "..." you can see the permissions, it's executable now.